zookeeper-server/src/main/java/org/apache/zookeeper/server/admin/Commands.java

```diff
@@ -657,11 +647,31 @@ public CommandResponse run(ZooKeeperServer zkServer, Map<String, String> kwargs)
                                 TreeMap::new));
             }
 
+            private String getMultiAddressString(QuorumPeer.QuorumServer qs) {
+                return qs.addr.getAllAddresses().stream()
+                        .map(address -> getSingleAddressString(qs, address))
+                        .collect(Collectors.joining(","));
+            }
+
+            private String getSingleAddressString(QuorumPeer.QuorumServer qs, InetSocketAddress address) {
```

Comment:
I believe that we don't need this magic here which is trying to recreate the original config file contents. The output of admin command doesn't need to match with the configuration, so you can just dump the contents of the internal representation (map) which I think more helpful for anyway reading this output.

Reply:
Yes, I agree, I will change it. 

This admin command is about to show the internal view of the voting members (how the zookeeper server thinks who the voting members are and where do they listen). I wouldn't complicate this PR any further, but it might be a good idea to create a follow-up ticket to have some admin command showing that if the given server can reach all the different ports of the other servers (not only the voting members). It can help debugging network problems, showing if certain network interfaces on some servers are unreachable.

Reply:
I changed the format, this is how the admin command responds now (I haven't push the commit yet):
```
{
  "current_config" : {
    "1" : {
      "server_addresses" : [ "/172.16.101.11:2888", "/172.16.102.11:2888" ],
      "election_addresses" : [ "/172.16.101.11:3888", "/172.16.102.11:3888" ],
      "client_address" : "/0.0.0.0:2181",
      "learner_type" : "participant"
    },
    "2" : {
      "server_addresses" : [ "/172.16.101.22:2888", "/172.16.102.22:2888" ],
      "election_addresses" : [ "/172.16.101.22:3888", "/172.16.102.22:3888" ],
      "client_address" : "/0.0.0.0:2181",
      "learner_type" : "participant"
    },
    "3" : {
      "server_addresses" : [ "/172.16.101.33:2888", "/172.16.102.33:2888" ],
      "election_addresses" : [ "/172.16.101.33:3888", "/172.16.102.33:3888" ],
      "client_address" : "/0.0.0.0:2181",
      "learner_type" : "participant"
    }
  },
  "command" : "voting_view",
  "error" : null
}
```

Reply:
Perfect!